package/resource_monitor/resource_monitor/src/query_sockets.c

```diff
@@ -761,7 +761,7 @@ static bool query_sockets_prepare(u16 *msg_type, u8 *len, u8 *sbp_buf, void *con
   default: PK_LOG_ANNO(LOG_ERR, "Invalid state value: %d", state->send_state);
   }
 
-  switch (state->send_state) {
+  switch (current_state) {
```

Comment:
curious why this slipped in. seems unrelated?

Reply:
It's a bugfix for the query_sockets module which was only reporting the top-9 sockets users instead of the top-10-- can split into separate PR if needed

Reply:
ahh, missed that. nope! can definitely leave in, just a note in the PR/squash would be good